escrow/free_call_payment_handler.go

```diff
@@ -84,3 +96,59 @@ func (h *freeCallPaymentHandler) CompleteAfterError(payment handler.Payment, res
 	return nil
 }
 
+func checkIfFreeCallsAreAllowed(username string) (allowed bool, err error) {
+	response,err := sendRequest(nil,config.GetString(config.MeteringEndPoint)+"/usage/freecalls",username)
+	return checkResponse(response)
+}
+
+//Set all the headers before publishing
+func sendRequest(json []byte, serviceURL string,username string) (*http.Response, error) {
+	req, err := http.NewRequest("GET", serviceURL, bytes.NewBuffer(json))
```

Comment:
Can you please make sure that the services are called on https?

Reply:
the URL is https